### PR TITLE
- Added warnings pragma as recommended by CPANTS.

### PR DESCRIPTION
**Pull request by manwar**
Hi,

Please review the PR.

Many Thanks.
Best Regards,
Mohammad S Anwar

### CODE REVIEW

**Comment by oalders** [comment]
Hi @manwar.  Thanks for this!

Should we add `Test::FailWarnings` to the test suite to ensure that we're not introducing any new warnings?

**Comment by manwar** [comment]
Hi @oalders 

That's brilliant idea and good catch.

Best Regards,
Mohammad S Anwar

**Comment by oalders** [comment]
@manwar I apologize for losing track of this PR.  Would you be able to resolve the conflicts and force-push your commits again?

**Comment by manwar** [comment]
Sure in a moment

File: lib/HTTP/Cookies/Microsoft.pm [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package HTTP::Cookies::Microsoft;
 
 use strict;
+use warnings;
+use vars qw(@ISA);
```
Comment (JRaspass) [inline lib/HTTP/Cookies/Microsoft.pm:5]: Why's the legacy `vars` pragma being introduced? `@ISA` is declared correctly with `our` a few lines down. Though something like `parent` might be better still it's out of the scope of this PR.

**Comment by manwar** [comment]
> @manwar is the feedback something you'd like to address? I promise to keep track of the PR this time. 😢

Sure, I will look into this weekend, promise.

**Comment by manwar** [comment]
This is all messed up. I would rather create fresh pull request. Please ignore this pull request.

**Pull request closed without merging**